--- cpprestsdk-2.10.19/Release/cmake/cpprest_find_boost.cmake.orig	2026-05-16 19:17:07.871660659 +0200
+++ cpprestsdk-2.10.19/Release/cmake/cpprest_find_boost.cmake	2026-05-16 19:17:41.148147052 +0200
@@ -27,11 +27,11 @@ function(cpprest_find_boost)
     if (EXISTS "${PROJECT_SOURCE_DIR}/../Build_iOS/boost")
       set(IOS_SOURCE_DIR "${PROJECT_SOURCE_DIR}/../Build_iOS")
       set(Boost_LIBRARIES "${IOS_SOURCE_DIR}/boost/lib" CACHE INTERNAL "")
-      set(Boost_INCLUDE_DIR "${IOS_SOURCE_DIR}/boost/include" CACHE INTERNAL "")
+      set(Boost_INCLUDE_DIRS "${IOS_SOURCE_DIR}/boost/include" CACHE INTERNAL "")
     else()
       set(IOS_SOURCE_DIR "${PROJECT_SOURCE_DIR}/../Build_iOS")
       set(Boost_LIBRARIES "${IOS_SOURCE_DIR}/boost.framework/boost" CACHE INTERNAL "")
-      set(Boost_INCLUDE_DIR "${IOS_SOURCE_DIR}/boost.framework/Headers" CACHE INTERNAL "")
+      set(Boost_INCLUDE_DIRS "${IOS_SOURCE_DIR}/boost.framework/Headers" CACHE INTERNAL "")
     endif()
   elseif(ANDROID)
     set(Boost_COMPILER "-clang")
@@ -54,7 +54,7 @@ function(cpprest_find_boost)
   add_library(cpprestsdk_boost_internal INTERFACE)
   # FindBoost continually breaks imported targets whenever boost updates.
   if(1)
-    target_include_directories(cpprestsdk_boost_internal INTERFACE "$<BUILD_INTERFACE:${Boost_INCLUDE_DIR}>")
+    target_include_directories(cpprestsdk_boost_internal INTERFACE "$<BUILD_INTERFACE:${Boost_INCLUDE_DIRS}>")
     set(_prev)
     set(_libs)
     foreach(_lib ${Boost_LIBRARIES})
--- cpprestsdk-2.10.19/Release/CMakeLists.txt.orig	2026-05-16 21:19:08.095336937 +0200
+++ cpprestsdk-2.10.19/Release/CMakeLists.txt	2026-05-16 21:19:25.855240723 +0200
@@ -168,10 +168,7 @@ elseif(CMAKE_CXX_COMPILER_ID MATCHES "GN
   set(WARNINGS -Wall -Wextra -Wunused-parameter -Wcast-align -Wcast-qual -Wconversion -Wformat=2 -Winit-self -Winvalid-pch -Wmissing-format-attribute -Wmissing-include-dirs -Wpacked -Wredundant-decls -Wunreachable-code)
   set(LD_FLAGS "${LD_FLAGS} -Wl,-z,defs")
 
-  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -fno-strict-aliasing")
-  if(CMAKE_CXX_COMPILER_VERSION VERSION_LESS 4.8)
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fpermissive -D_GLIBCXX_USE_SCHED_YIELD -D_GLIBCXX_USE_NANOSLEEP")
-  endif()
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++14 -fno-strict-aliasing")
 
 elseif(CMAKE_CXX_COMPILER_ID MATCHES "MSVC")
   message("-- Setting msvc options")
--- cpprestsdk-2.10.19/Release/tests/functional/pplx/pplx_test/pplx_op_test.cpp.orig	2026-05-16 21:39:17.785450149 +0200
+++ cpprestsdk-2.10.19/Release/tests/functional/pplx/pplx_test/pplx_op_test.cpp	2026-05-16 21:39:53.685255663 +0200
@@ -57,7 +57,7 @@ class pplx_dflt_scheduler : public pplx:
     virtual void schedule(pplx::TaskProc_t proc, void* param)
     {
         pplx::details::atomic_increment(s_flag);
-        m_pool->service().post([=]() -> void { proc(param); });
+        boost::asio::post(m_pool->service(), [=]() -> void { proc(param); });
     }
 
 public:
